### ISSUE TITLE
Improve the design 

### ISSUE THREAD

**Issue by AdelBeit** [AdelBeit/dnk#2]
Looking for a designer or a creative to come up with a better looking UI with a nice color scheme
This contribution doesn't need to be in the form of code, I'm looking for a figma file or something similar, the implementation can be a separate concern

the figma file is published here
https://www.figma.com/community/file/1166522358464320123

if you'd like to contribute remix the file and post the link in the comments here for review :)

**Comment by Vidhanjain143** [comment]
Assign this to me.

**Comment by YuriCodes** [comment]
Hi! I can help make a color palette that fits the app better and will help with accessibility concerns

**Comment by Khushi-design-20** [comment]
Hey, just looked at your Figma file, I would love to contribute some design changes.

**Comment by AdelBeit** [comment]
I just went through a design iteration, I had to leave the neumorphic style behind because I couldn't figure out how to make it look better and improve accessibility. That style just is known for having poor accessibility.
I'm not super happy with the new design either but it was what I could come up with for the time being to at least it's more accessible. 
I've updated the repo with the published figma file for anyone to remix. 
posting the link here as well
https://www.figma.com/community/file/1166522358464320123

Feel free to come up with something new or improve the existing design and post your figma link here for me to see it!

I went ahead and assigned you both to the issue, if you change your mind at any point feel free to unassign yourself :)
